### PR TITLE
Update README.md for sample FastAPI

### PR DESCRIPTION
**Pull request by AndrewNgo-ini**
### Describe the changes you have made:
The current sample does not work so I fix it.
### Reference any relevant issue (Fixes #000)

- [x ] I have performed a self-review of my code:

### I have tested the code on the following OS:
- [ ] Windows
- [ ] MacOS
- [ x] Linux

### AI Language Model (if applicable)
- [ ] GPT4
- [ ] GPT3
- [ ] Llama 7B
- [ ] Llama 13B
- [ ] Llama 34B
- [ ] Huggingface model (Please specify which one)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,14 +320,18 @@ The generator update enables Open Interpreter to be controlled via HTTP REST end
...
 app = FastAPI()
 
 @app.get("/chat")
-def chat_endpoint(message):
-    return Response(interpreter.chat(message, stream=True), media_type="text/event-stream")
+def chat_endpoint(message: str):
+    def event_stream():
+        for result in interpreter.chat(message, stream=True):
+            yield f"data: {result}\n\n"
+    return StreamingResponse(event_stream(), media_type="text/event-stream")
```
Comment (ericrallen) [inline README.md:334]: This is a minor nitpick, but can we put a line break before this `return` so it’s easier for the reader to scan and parse the whole handler? Or maybe move the `event_stream` function declaration out of the `chat_endpoint` handler?

**Comment by ericrallen** [comment]
Hey there, @AndrewNgo-ini!

Thanks for updating this!

I left one minor comment about readability.

**Review by ericrallen — APPROVED** [review]
Thanks!

**Comment by KillianLucas** [comment]
Nice catch! Thanks @AndrewNgo-ini and @ericrallen for reviewing.

**Pull request merged**